tests/test_image.py

```diff
@@ -112,14 +113,14 @@ def testImShiftStack(parity, dtype):
     im2 = ims.shift(shifts.astype(dtype))
     # ground truth numpy roll
     im3 = np.array(
-        [np.roll(ims_np[i, :, :], -shifts[i], axis=(0, 1)) for i in range(n)]
```

Comment:
Can you add a comment here explaining the axis swap?

Reply:
Sure! Done.